### DISCUSSION TITLE
Can the result be correct?

### DISCUSSION THREAD

**Discussion by nepomuk16321** [eyeonus/Trade-Dangerous#171] [Q&A] [answered]
Please take a look at the two results after executing “RUN”. The first run with 10,000,000 million and the second run with 100,000,000 million. I think something is wrong.
![image](https://github.com/eyeonus/Trade-Dangerous/assets/20616775/bd9c4744-88f1-4f93-9cdd-7bf2002c15ab)

**Comment by kfsone** [comment]
You've told TD that you can jump 16.5 ly per hop, but it can travel a maximum of 22 lightyears on empty which means a maximum of 1.25 hops before it has to buy something, which may force its hand to picking something up from the first hop, and specifying a start station causes it to consider going to another station in the first system as a hop, and having only 2 hops left may preclude it finding a killer run for you.

**Comment by kfsone** [comment]
Also, an alternative to --empty 22 would be --start-jumps 1 (or --start-jumps 2). It can add to the run calculation time but it lets it find a better starting station before beginning the route.

**Reply by nepomuk16321** [reply]
I found the routes I chose or found to be OK. Unfortunately I couldn't fly them because I was shot down :-(.
I mean the yield after the 3 jumps, from 10 million at the start to 140 million at the end, if I'm interpreting that correctly. :-)

**Reply by eyeonus** [reply]
The run calculates a profit of ~130M. 10M + 130M = 140M

**Reply by nepomuk16321** [reply, marked as answer]
I looked at the stations on INARA. And the sticking point, that I hadn't expected, was the fleet carrier.
He would have sold me the tritium for about 600,000 per ton. So that's almost the 130 million profit.
So all right, I hadn't expected that.  :-)
